src/main/java/io/javalin/Javalin.java

```diff
@@ -106,6 +99,17 @@ public JavalinServlet servlet() {
         return this.servlet;
     }
 
+    /**
+     * Create the OpenApi Schema of this instance. Call this method after all the handler are registered.
+     * This requires that the "enableOpenApi" option is enabled otherwise an IllegalStateException is thrown.
+     */
+    public OpenAPI createOpenAPISchema() {
```

Comment:
Javalin uses strict camel casing, so `openAPI` should be `openApi`. That aside, can we get rid of this method? IMO it would be better if we could hand the relevant information to `Javalin.create(config)`, and then have the schema exposed on an endpoint. Why is the user required to call `createOpenAPISchema` themselves?

Reply:
The intention is that the user might want to modify the schema and provide his own endpoint (For example as a yaml file instead of a json). Or he wants to save the schema as a static file.
If I just provide the endpoint, I limit the generation to this specific use case.

But if you want, I can move it out of the Javalin Class into a static function.

Reply:
> But if you want, I can move it out of the Javalin Class into a static function.

That sounds like a good solution.